### DISCUSSION TITLE
'code' button is gone

### DISCUSSION THREAD

**Discussion by ghost** [community/community#57993] [Repositories] [answered]
### Select Topic Area

General

### Body

Hi

In the past each repository had a green 'code' button and when clicked you can download the repository as a ZIP file.
Why is it gone now?

**Comment by airtower-luna** [comment]
It's still there as far as I can see:

![grafik](https://github.com/community/community/assets/5814919/d4064922-5055-418f-b230-ba21fa9d9066)

**Comment by ghost** [comment, marked as answer]
Ok, I see it!

**Comment by viaSSH** [comment]
For anyone in same situation,

```
(for chrome browser) 
go to repository page -> press f12 -> toggle device toolbar -> page refresh -> toggle device toolbar once

toggle device toolbar = change mobile/web view
```
code button will be gone. 🧐
if you refresh page, you will see the button again.

**Reply by chickod** [reply]
This doesn't work
